.idea/spatialos-sdk-rs.iml

```diff
@@ -3,41 +3,42 @@
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/spatialos-sdk-sys/src" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/spatialos-sdk-sys\examples" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/spatialos-sdk-sys\tests" isTestSource="true" />
-      <sourceFolder url="file://$MODULE_DIR$/spatialos-sdk-sys\benches" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/spatialos-sdk-sys/examples" isTestSource="false" />
```

Comment:
Should we delete this file from the repo?

Reply:
I believe these are meant to be committed (at least from what I can see in the guidance).

It means if you open the project in an IntellJ you get a nicely configured set of run configs: 

![2020-05-08 00_09_33-spatialos-sdk-rs – download rs CLion](https://user-images.githubusercontent.com/13353733/81353157-3ffeb880-90c0-11ea-91e2-d99f1c28978a.png)

This was the first time this has appeared in a diff, I think its because I upgraded from CLion 2019.3 to 2020.1 recently. If it keeps thrashing, I'll purge it from the repo 😄